docs/source/en/model_doc/vitpose.md

```diff
@@ -89,8 +89,37 @@ pose_results = image_processor.post_process_pose_estimation(outputs, boxes=[pers
 image_pose_result = pose_results[0]  # results for first image
 ```
 
+### ViTPose++ models
+
+The best checkpoints are those of the [ViTPose++ paper](https://arxiv.org/abs/2212.04246). ViTPose++ models employ a so-called [Mixture-of-Experts (MoE)](https://huggingface.co/blog/moe) architecture for the ViT backbone, resulting in better performance. The ViTPose++ models require the `dataset_index` argument to be passed in the forward of the model, to indicate which experts to use for each example in the batch. Example usage is shown below:
```

Comment:
Maybe more practical to link to the [collection](https://huggingface.co/collections/usyd-community/vitpose-677fcfd0a0b2b5c8f79c4335) of checkpoints instead of the paper?